### ISSUE TITLE
Bottom row of toggles doesn't announce what the value is being picked for

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#309585]
Testing #309100

<img width="940" height="347" alt="Image" src="https://github.com/user-attachments/assets/a5a6da32-5afc-427f-9379-45658329d255" />

The top row (agent picker, model picker) describe what they are ("pick model") in addition to what they are set to.

The _bottom_ row only says what value it is set to.

I think we should announce what those dropdowns do in addition to what they are set to.

**Comment by meganrogge** [comment]
I am guessing I would've noticed this, maybe just an orca issue
